test/season_test.rb

```diff
@@ -13,31 +13,31 @@ def setup
   end
 
   def test_biggest_bust
-    assert_equal "Atlanta United", @stat_tracker.biggest_bust('20132014')
+    assert_equal 0, @stat_tracker.biggest_bust('20122013')
```

Comment:
i think you forgot to pull/merge to master and it is trying to change these back to a previous version